pkg/ui/workspaces/db-console/src/views/statements/activeStatementsSelectors.tsx

```diff
@@ -52,6 +52,22 @@ const sortSettingLocalSetting = new LocalSetting<AdminUIState, SortSetting>(
   { ascending: false, columnTitle: "startTime" },
 );
 
+// autoRefreshLocalSetting is shared between the Active Statements and Active
+// Transactions components.
+const autoRefreshLocalSetting = new LocalSetting<AdminUIState, boolean>(
+  "isAutoRefreshEnabled/ActiveExecutions",
+  (state: AdminUIState) => state.localSettings,
+  true,
+);
+
+// lastRefreshTimestampLocalSetting is shared between the Active Statements and
+// Active Transactions components.
+const lastRefreshTimestampLocalSetting = new LocalSetting<AdminUIState, string>(
```

Comment:
You don't need to create a new value for this. I think you can pull this timestamp from the sessions redux state (last updated) so that they can see the latest time the info was returned.